src/include/network/lan/cmdline.rb

```diff
@@ -191,31 +192,22 @@ def AddHandler(options)
         return false
       end
 
+      builder = Y2Network::InterfaceConfigBuilder.for(LanItems.type)
+      builder.name = LanItems.GetCurrentName()
+
       case LanItems.type
       when "bond"
-        LanItems.bond_slaves = options.fetch("slaves", "").split(" ")
+        builder["BOND_SLAVES"] = options.fetch("slaves", "").split(" ")
       when "vlan"
-        LanItems.vlan_etherdevice = options.fetch("ethdevice", "")
+        builder["ETHERDEVICE"] = options.fetch("ethdevice", "")
       when "br"
-        LanItems.bridge_ports = options.fetch("bridge_ports", "")
+        builder["BRIDGE_PORTS"] = options.fetch("bridge_ports", "")
       end
```

Comment:
why this case is not in `update_builder_from_options!` ?

Reply:
Well, basically because options where not allowed edition but just when adding. Although from my POV should be valid in both cases and the behavior could be also moved to the convenience method.